components/dashboard/TextArea/index.tsx

```diff
@@ -49,14 +50,33 @@ const TextArea = ({ shiftRight, setShiftRight }: TextAreaProps) => {
       ).withConverter(notesConverter)
   );
 
+  const addToStore1 = async (
+    key: string,
+    value: { editorTitle: string; editorContent: string }
+  ) => {
+    set(key, value);
+  };
+
+  const getFromStore1 = async (selectedNoteId: string) => {
+    if (!selectedNoteId) return;
+    return (
+      get(selectedNoteId) || {
+        editorTitle: "Untitled",
+        editorContent: "",
+      }
+    );
+  };
+
   const saveNoteChanges = async (noteId: string) => {
-    const lsInput = localStorage.getItem("editorContent");
-    const lsTitle = localStorage.getItem("editorTitle");
-    if (!isSynced && selectedNoteId && lsInput && lsTitle) {
+    if (!selectedNoteId) return;
```

Comment:
Should specify a return type.

Reply:
that function is not supposed to return anything, so void?

Reply:
null is preferable.

Reply:
Wait, this is the saveNoteChanges feature. Sorry, no return type needed here.